registration.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+    // functions to use for registration
+    
+    function registerForOfflineEvent($event){
+        $phone = $_POST['phone'];
+        $mail = $_POST['mail'];
+        $name = $_POST['name'];
+        $studiengang = $_POST['studiengang'];
+        $semester = $_POST['semester'];
+        $abschluss = $_POST['abschluss'];
+
+        if(empty($phone) || empty($mail) || empty($name) || empty($studiengang) || empty($semester) || empty($abschluss)){
+            echo "<div class='block info'>Fehler. Du hast nicht alle erforderlichen Daten angegeben.</div>";
+            return;
+        }
+
+
+        $file = $event["path"];
+
+        // already registered
+        if(strpos(file_get_contents($file), $mail) !== false || strpos(file_get_contents($file), $phone) !== false){
+            echo "<div class='block info'>Du bist zu dieser Veranstaltung bereits angemeldet.</div>";
```

Comment:
Didn't check how this looks but a class `block error` might be helpful (this looks like it is an error even though the message doesn't contain "Fehler" like above).
(Since there are a lot of other error messages.)